packages/cache/src/internal/cacheHttpClient.ts

```diff
@@ -110,75 +85,6 @@ export function getCacheVersion(
     .digest('hex')
 }
 
-export async function retry<T>(
```

Comment:
Unfortunately, I think this is a breaking change. If a user was importing `retry` or any other exported function from this file, their imports will break as they have to now import from `downloadUtils` or `requestUtils`

This change makes sense to me though, and I like the extra testing it allows. We can bump the major version and list these breaking changes in the release.

Reply:
Bumped the version to `1.0.0`